tests/e2e/hcp_machine_pool_test.go

```diff
@@ -26,54 +103,244 @@ var _ = Describe("Create Machine Pool", labels.Feature.Machinepool, func() {
+			downReplicas := 1
+			scaleNodePool(mpName, "down", downReplicas, true)
+
+			By("Scale down a machine pool replicas to 0")
+			zeroReplica := 0
+			scaleNodePool(mpName, "down", zeroReplica, true)
+		},
+		Entry("For amd64 cpu architecture [id:60278]", constants.M5XLarge, constants.AMD),
+		Entry("For arm64 cpu architecture [id:60278]", constants.M6gXLarge, constants.ARM),
+	)
 
-			By("Create another machinepool without security groups and describe it")
-			mpName = "mp-72195-nsg"
-			_, err = rosaClient.MachinePool.CreateMachinePool(clusterID, mpName,
-				"--replicas", "1",
+	DescribeTable("Scale up/down a machine pool with invalid replica", labels.Critical, labels.Runtime.Day2,
+		func(instanceType string, updatedReplicas string, expectedErrMsg string) {
+			By("Create machinepool with instance " + instanceType)
+			mpName := common.GenerateRandomName("mp-60278", 2)
+			desiredReplicas := 1
+			_, err := rosaClient.MachinePool.CreateMachinePool(clusterID, mpName,
+				"--instance-type", instanceType,
+				"--replicas", fmt.Sprintf("%v", desiredReplicas),
 				"-y",
 			)
 			Expect(err).ToNot(HaveOccurred())
 			defer rosaClient.MachinePool.DeleteMachinePool(clusterID, mpName)
-			By("Check the machinepool detail by describe")
-			mpDescription, err = rosaClient.MachinePool.DescribeAndReflectNodePool(clusterID, mpName)
-			Expect(err).ToNot(HaveOccurred())
-			Expect(mpDescription.AdditionalSecurityGroupIDs).To(BeEmpty())
+
+			_, err = rosaClient.MachinePool.EditMachinePool(clusterID, mpName,
+				"--replicas", fmt.Sprintf("%v", updatedReplicas),
+				"-y",
+			)
+			Expect(err.Error()).Should(ContainSubstring(expectedErrMsg))
+		},
+
+		Entry("Scale replica too large [id:60278]", constants.M52XLarge, "1000", "exceeds the maximum allowed"),
+		Entry("Scale replica to -1 [id:60278]", constants.M52XLarge, "-1", "Replicas must be a non-negative number"),
+		Entry("Scale replica to a char [id:60278]", constants.M52XLarge, "a", "invalid syntax"),
+	)
+
+	Describe("Scale up/down a machine pool enabling autoscale", func() {
+		When("with the valid min replica and max replica", func() {
+			It("should succeed to scale up/down the machine pool[id:60278]", labels.Medium, labels.Runtime.Day2, func() {
+				instanceType := constants.M52XLarge
+				By("Create machinepool with " + " instance " + instanceType + " and enable autoscale")
+
+				mpPrefix := "autoscale"
+				mpName := common.GenerateRandomName(mpPrefix, 2)
+				minReplica := 1
+				maxReplica := 3
+				_, err := rosaClient.MachinePool.CreateMachinePool(clusterID, mpName,
+					"--instance-type", instanceType,
+					"--enable-autoscaling",
+					"--min-replicas", fmt.Sprintf("%v", minReplica),
+					"--max-replicas", fmt.Sprintf("%v", maxReplica),
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+				defer rosaClient.MachinePool.DeleteMachinePool(clusterID, mpName)
+
+				// TODO There's issue here https://issues.redhat.com/browse/OCM-10147
```

Comment:
We don't include OCM jira issue in code.

Reply:
OK, I'll replace the link with jira issue ID, which I can know here still has some issue.

Reply:
Why not include the case ID in the jira card of the bug? So when you verify the bug you can update the script accordingly to the fixing.